pkg/overlay/overlay.go

```diff
@@ -15,6 +15,8 @@ import (
 	"github.com/containers/storage/pkg/system"
 	"github.com/containers/storage/pkg/unshare"
 	"github.com/opencontainers/runtime-spec/specs-go"
+	"github.com/opencontainers/selinux/go-selinux/label"
+	"github.com/pkg/errors"
```

Comment:
I'm not sure how/why this line was dragged in.  I don't see an additional call to errors here in the changes.

Reply:
I believe this was [addressed in a later commit/push](https://github.com/containers/buildah/pull/5955/files#diff-97a773bb4375bb0de346986ec764abdbc2f07b74a298953de1973eb747ed9759L11).